### ISSUE TITLE
Kotlin compilation error for function 'onRequestPermissionsResult' in beta channel

### ISSUE THREAD

**Issue by maximletushov** [QuisApp/flutter_contacts#49]
Hi,

### Console output
Running Gradle task 'assembleDebug'...

e: /dir/.pub-cache/hosted/pub.dartlang.org/flutter_contacts-1.1.3/android/src/main/kotlin/co/quis/flutter_contacts/**FlutterContactsPlugin.kt**: (27, 8): Class 'FlutterContactsPlugin' is not abstract and **does not implement** abstract member public abstract **fun onRequestPermissionsResult**(p0: Int, p1: Array<(out) String!>, p2: IntArray): Boolean **defined in** io.flutter.plugin.common.PluginRegistry.RequestPermissionsResultListener

e: dir/.pub-cache/hosted/pub.dartlang.org/flutter_contacts-1.1.3/android/src/main/kotlin/co/quis/flutter_contacts/**FlutterContactsPlugin.kt**: (137, 5): '**onRequestPermissionsResult**' **overrides nothing**

**FAILURE: Build failed with an exception.**
What went wrong:
Execution failed for task ':flutter_contacts:compileDebugKotlin'.

### Found solution
Here is current signature of function onRequestPermissionsResult.

```
override fun onRequestPermissionsResult(
    requestCode: Int,
    permissions: Array<out String>?,
    grantResults: IntArray?
): Boolean

```

**WHEN I CHANGE PARAMETERS TO NON-NULLABLE THEN IT BUILDS SUCCESSFULLY.**
```
override fun onRequestPermissionsResult(
    requestCode: Int,
    permissions: Array<out String>,
    grantResults: IntArray
): Boolean

```

Please use this fix if it's correct or please guide me how to solve issue in the right way. **Thanks.**

### P.S. Additional info
Settings of my app:

_pubspec.yaml_
flutter_contacts: 1.1.3

_build.gradle_
ext.kotlin_version = '1.6.10'
classpath 'com.android.tools.build:gradle:7.1.2'

_gradle-wrapper.properties_
distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.1-all.zip

**Comment by rahmanrezaee** [comment]
I also face this issue in flutter 2.10.3

**Comment by joachim-quis** [comment]
Thanks for flagging and debugging! Applied your fix in [version 1.1.4](https://pub.dev/packages/flutter_contacts/versions/1.1.4).

**Comment by maximletushov** [comment]
> Thanks for flagging and debugging! Applied your fix in [version 1.1.4](https://pub.dev/packages/flutter_contacts/versions/1.1.4).

Thanks 👍
